test/test_add_data.py

```diff
@@ -28,14 +28,13 @@ def test_add_all_data():
 
 
 def test_add_node_data():
-
     grid = two_triangles()
 
     # Create a UVar object for the depths:
     depths = UVar('depth', location='node', data=[1.0, 2.0, 3.0, 4.0])
     depths.attributes['units'] = 'm'
-    depths.attributes["standard_name"] = "sea_floor_depth"
-    depths.attributes["positive"] = "down"
+    depths.attributes['standard_name'] = 'sea_floor_depth'
```

Comment:
> so what's the rule for this project?

I don't have any strong feelings about this.  I only changed because I write a lot of JSON (where I need the double quotes) and python strings (where they don't matter) in a few of my projects.  So I avoid mixing them to make it clear in my head when.  I can revert those if you want.